react-table/src/Table.tsx

```diff
@@ -157,6 +161,7 @@ function Cell<T>(props: ICellProps<T>) {
         <td
             style={css}
```

Comment:
is it worth adding a conditional style for `OnDragStart !== undefined` to make the mouse symbol change if it is dragable?

Reply:
Added the symbol change on line 155